demo/wp_nd.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# Note: This demo is a repeat of wp_2d, but using WaveletPacketND instead
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from pywt import WaveletPacketND
+import pywt.data
+
+
+arr = pywt.data.aero()
+
+maxlevel = 2
+wp2 = WaveletPacketND(arr, 'db2', 'symmetric', maxlevel=maxlevel)
+
+# Show original figure
+plt.imshow(arr, interpolation="nearest", cmap=plt.cm.gray)
+
+fig = plt.figure()
+i = 1
+nsubplots = len(wp2.get_level(maxlevel, 'natural'))
+nrows = int(np.floor(np.sqrt(nsubplots)))
+ncols = int(np.ceil(nsubplots/nrows))
+for node in wp2.get_level(maxlevel, 'natural'):
+    ax = fig.add_subplot(nrows, ncols, i)
+    ax.set_title("%s" % (node.pretty_path, ))
```

Comment:
`pretty_path` is not an attribute. should it just be `path`? Or `path_tuple`?

Reply:
I ran the demo, this was meant to be `path_tuple` I believe - made that fix.

Reply:
thanks, that is probably correct. I will take a quick look again.